app/src/main/java/com/sdp/movemeet/view/profile/ProfileActivity.java

```diff
@@ -116,45 +129,61 @@ public boolean onNavigationItemSelected(@NonNull MenuItem menuItem) {
         switch (menuItem.getItemId()) {
             case R.id.nav_home:
                 Navigation.goToHome(this.navigationView);
+                finish();
                 break;
             case R.id.nav_edit_profile:
+                finish();
```

Comment:
You need to remove the finish() when it try to navigate on the page where we already are. But all this will be remove by the usage of the new Navigation class

Reply:
Ok ok! Yes, I will merge my branch onto our main branch and this Navigation class will be added later on.